src/frontend/apps/lti_site/apps/deposit/components/Dashboard/DashboardInstructor/index.tsx

```diff
@@ -1,17 +1,22 @@
-import { Select } from '@openfun/cunningham-react';
-import { Box, Pagination } from 'grommet';
+import { Pagination, Select } from '@openfun/cunningham-react';
+import { Box } from 'grommet';
 import { Maybe } from 'lib-common';
 import { BoxLoader, FileDepository, Heading, Text } from 'lib-components';
 import React, { FocusEvent, useState } from 'react';
 import { FormattedMessage, defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
 
 import { DepositedFileRow } from 'apps/deposit/components/Dashboard/common/DepositedFileRow';
 import {
   useDepositedFiles,
   useUpdateFileDepository,
 } from 'apps/deposit/data/queries';
 
-type SelectProps = React.ComponentPropsWithoutRef<typeof Select>;
+const BoxPagination = styled(Box)`
+  .c__pagination__goto {
+    display: none;
+  }
+`;
```

Comment:
Maybe we could propose to cunninham to add a prop on Pagination for hidding this element without using custom css ?

Reply:
Yes: https://github.com/openfun/cunningham/issues/183